projects/extension/sql/idempotent/012-vectorizer-int.sql

```diff
@@ -405,7 +405,10 @@ declare
 begin
     -- create the table
     select pg_catalog.format
-    ( $sql$create table %I.%I(%s, queued_at timestamptz not null default now())$sql$
+    ( $sql$create table %I.%I(%s
+    , queued_at pg_catalog.timestamptz not null default now()
+    , retries pg_catalog.int4 default 0
+    , retry_after pg_catalog.timestamptz)$sql$
```

Comment:
```suggestion
    ( $sql$
      create table %I.%I
      ( %s
      , queued_at pg_catalog.timestamptz not null default now()
      , retries pg_catalog.int4 default 0
      , retry_after pg_catalog.timestamptz
      )
      $sql$
```

incredibly nit picky. sorry.